lib/hooks/useFocusIt.tsx

```diff
@@ -0,0 +1,38 @@
+import { useEffect, useRef } from "react";
+
+/**
+ * Utility function to focus an element on load. This helps manage the users focus
+ * e.g. on a page update, set the focus to the heading.
+ *
+ * @param elRef React element to focus
+ * @param addTabindIndex Whether to add a tab-index attribute. If true (default), a
+ *    `tab-index=-1` attribute is is automatically added to the element if none exist.
+ */
+export const useFocusIt = ({
```

Comment:
Definitely not how Hooks were really intended (sharing state + stateful logic). Though I can't think of a better way to tap into the React life-cycle and do this as a util method